### PR TITLE
JAMES-2507 3.1.0 Release

### PR DESCRIPTION
**Pull request by chibenwa**
Tweet:

```
Apache James 3.1.0 is out! 

Check it out here: {link_to_the_post}

Many thanks to all contributors!
```

Call for vote:

```
To: server-dev@james.apache.org
Subject: Call for vote: Apache James 3.1.0

Hi,

I would like to propose the 3.1.0 release of the Apache James server.

Furthermore, you can see changes proposed to the website at the occasion of that release, as well as communication on this Github pull request: {link to future Apache PR}.

You can find:

 - The maven release staged in http://repository.apache.org/ as artifact #1036
 - 3.1.0 sources and binaries are uploaded on Apache dist. Please note that we included JPA Guice and JPA SMTP guice.

Voting rules:
 - This is a majority approval: this release may not be vetoed.
 - A quorum of 3 binding votes is required
 - The votes starts at [X]
 - The votes ends at [X + 1 week]

You can answer to it just with +1 and -1. Down-votes may be motivated.

Cheers,

Benoit Tellier
```

Announce:

```
Subject: [ANNOUNCE] Apache James 3.1.0 released
To: announce@apache.org, server-user, server-dev

The Apache James community is proud to announce you the 3.1.0 release of the Apache James server. 

Apache James provides a complete, stable, secure and extendable Mail Servers running on the JVM. More details about Apache James can be found at: https://james.apache.org/

The release is available for download at: https://james.apache.org/download.cgi#Apache_James_Server

Changes from 3.1.0 includes:

 - Move to Java 8
 - Improve Mail Repositories handling, including a nice web API
 - Improve Mail Queues handling, including a nice web API
 - Improve RRT (Recipient Rewrite Table) implementation
 - Quota handling improvments, and in particular users can receive an email when they are near the limit of their quota
 - Many performances enhancement, in particular on Cassandra backend
 - Documentation updates
 - Delegating folders (full JMAP support, partial IMAP support)
 - Introduce an object store
 - Configurable mailbox listeners
 - MDN (Message Disposition notification) parsing and handling
 - SpamAssassin support with per user reports
 - Search in attachments
 - Data Leak Prevention
 - JPA SMTP Guice product
 - Cassandra migration process
 - Structured logging
 - RPM packaging (in addition to deb packaging)

We would like to thank all of contributors who made the release possible and
to invite others interested in helping out to engage the community on the dev
and users lists!
```

### CODE REVIEW

File: src/site/xdoc/server/quick-start.xml [hunk 1]
```diff
@@ -36,24 +36,24 @@
 Step 0: Requirements
 ####################
 
-  * JRE 1.6+
+  * JRE 1.8+
```
Comment (mbaechler) [inline src/site/xdoc/server/quick-start.xml]: we should remove the "+" as it probably doesn't work

File: src/site/xdoc/server/install.xml [hunk 2]
```diff
@@ -29,13 +29,11 @@
 
   <subsection name="Java Virtual Machine">
   
-    <p>James 3.0 requires a Java Runtime Environment. Java version 1.6 or higher is required to run the
+    <p>James 3.1 requires a Java Runtime Environment. Java version 1.8 or higher is required to run the
```
Comment (mbaechler) [inline src/site/xdoc/server/install.xml]: we should remove "or higher"

File: src/site/xdoc/server/install.xml [hunk 3]
```diff
@@ -29,13 +29,11 @@
...
        James application. The exact JREs available depends on the platform. In addition, the environment variable JAVA_HOME should be set to
        the JRE home directory before running James.</p>
-
-    <p>After James 3.0 release, James requires Java version 1.8 or higher.</p>
     
-    <p>James 3.0 has been successfully tested on OpenJDK, Sun JDK and IBM JDK</p>
+    <p>James 3.1 has been successfully tested on OpenJDK, Sun JDK and IBM JDK</p>
```
Comment (mbaechler) [inline src/site/xdoc/server/install.xml]: I doubt we did

File: src/homepage/_posts/2018-07-31-james-3.1.0.markdown [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Apache James Server 3.1.0"
+date:   2018-07-31 16:13:22 +0200
+categories: james update
+---
+
+James server 3.1.0 had been releases.
```
Comment (mbaechler) [inline src/homepage/_posts/2018-07-31-james-3.1.0.markdown]: has been ?

File: src/homepage/_posts/2018-07-31-james-3.1.0.markdown [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Here are some points w worked on:
```
Comment (mbaechler) [inline src/homepage/_posts/2018-07-31-james-3.1.0.markdown]: s/w/we

**Comment by mbaechler** [comment]
In announce : 

> Apache Jamesh provides a complete, stable, secure and extendable Mail Servers running on the JVM. More details about Apache James can be found at: https://james.apache.org/

s/Jamesh/James

**Comment by chibenwa** [comment]
Promoted to an Apache PR, and voting process just started.

**Pull request closed without merging**